cfgov/apache/conf.d/redirects.conf

```diff
@@ -508,8 +508,8 @@ RedirectMatch permanent ^\/jobs\/detail\/.*  /careers/current-openings/
 RedirectMatch permanent ^\/jobs\/(?!supervision)(?!technology-innovation-fellows)(?!jobs.json).* /about-us/careers/
 
 
-Redirect permanent /advisory-groups/advisory-groups-meeting-details/ /about-us/advisory-groups/
-Redirect permanent /advisory-groups/ /about-us/advisory-groups/
+Redirect permanent /advisory-groups/advisory-groups-meeting-details/ /rules-policy/advisory-committees/
+Redirect permanent /advisory-groups/ /rules-policy/advisory-committees/
```

Comment:
We'll need the following as well for any existing links/cached directs (line 512 is identical)

```suggestion
Redirect permanent /advisory-groups/ /rules-policy/advisory-committees/
Redirect permanent /about-us/advisory-groups/ /rules-policy/advisory-committees/
```

Reply:
Good catch. Fixed and rebased.